app/models/claim.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# A claim is an employee tribunal claim (form the ET1 form)
+class Claim < ApplicationRecord
+  has_many :claim_claimants, dependent: :destroy
+  has_many :claim_respondents, dependent: :destroy
+  has_many :claim_representatives, dependent: :destroy
+  has_many :claim_uploaded_files, dependent: :destroy
+
+  has_many :claimants, through: :claim_claimants
```

Comment:
@garytaylor why are you using has many with through? If you use just has_and_belongs_to_many you don't have to create the 'through' model. Or am I missing something?

Reply:
They are easier for the admin to manage and they work better with propagating 'touches'.  I also find it easier to use has_many :through to start with and remove the ones that  really aren't needed

Reply:
I lean towards @zaparka on this. Perhaps missing parts of the app design, prevent us from understanding why the `has_many :through` would be better option than `has_and_belongs_to_many`, which looks a much easier and viable option (no need to have separate models for the joins).

I quote best practice for such cases from the [Rails documentation](http://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many):
>The simplest rule of thumb is that you should set up a has_many :through relationship if you need to work with the relationship model as an independent entity. If you don't need to do anything with the relationship model, it may be simpler to set up a has_and_belongs_to_many relationship

Reply:
There are parts of the database model that I have missed (purposely as not required right now but will be in the future) where attributes on the join model are required.  Rather than do a detailed database design now in a typical non agile way, I think it is easier to start off with these join models and if no extra functionality / attributes within the join are required at the end, simply remove the model, do a has_and_belongs_to_many and specify the join table.  Doing it the other way around is harder.  Just a personal preference - but will iron itself out in the end.

Reply:
Ok, I see. Fine by me to keep it as is for now as we will have another chance to review it before we merge it on master branch.